gobblin-dataset-config/src/main/java/gobblin/config/client/ConfigClient.java

```diff
@@ -0,0 +1,17 @@
+package gobblin.config.client;
+
+import java.net.URI;
+
+import com.typesafe.config.Config;
+
+import gobblin.config.configstore.ConfigStore;
+
+public interface ConfigClient {
```

Comment:
I don't understand. What happened to the ConfigAccessor? What about versions? Why do clients need to know about ConfigStores?